diagnostic.md

```diff
@@ -57,13 +71,13 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+$('body').hide()
```

Comment:
Hey so this is again when we were asking you to traverse up from a low node in the tree. If this is still confusing let one of us know and we can review it.